File: vue/src/store/aa02.ts
import { defineStore } from 'pinia'

import {deleteAa02, getAa02List, writeAa02, getAa02SelectCode,Page} from "../api";

export interface aa02Type {
    aa02PK: Aa02Pk
    useyn: String
    sortOrder: String
    attr: String
    cname: String
    attr1: String
    code1: String
    desc1: String
    attr2: String
    code2: String
    desc2: String
    attr3: String
    code3: String
    desc3: String
    attr4: String
    code4: String
    desc4: String
    attr5: String
    code5: String
    desc5: String
    addUser: String
    addDate: Date
    addIp: String
    updateUser: String
    updateDate: Date
    updateIp: String
    bigo: String
}
export interface Aa02Pk {
    gubn: String
    code: String

}
export const Aa02Store = defineStore("Aa02Store",
    {
        state: () => ({
            aa02List:[] as aa02Type[],
            code:{
                BA0103:[] as aa02Type[],
                BA0105:[] as aa02Type[]
            },
        }),
        getters: {


        },
        actions: {
            getAa02List(params : Page) {
                return getAa02List(params)
            },
            delete(params : Aa02Pk[]) {
                return deleteAa02(params)
            },
            write(params: aa02Type) {
                return writeAa02(params)
            },
            getCode(gubn:String) {
                const result=getAa02SelectCode(gubn)
                result.then((response)=>{
                    // @ts-ignore
                    this.code[gubn]=response
                }).catch((error)=>{
                    console.log(error)
                })
            }

        }
    })

File: src/main/java/com/dipol/pdfconvert/data/entity/Aa02.java
package com.dipol.pdfconvert.data.entity;

import lombok.Data;

import javax.persistence.*;

import java.util.Date;

@Entity
@Data
public class Aa02 {
    @EmbeddedId
    private Aa02PK aa02PK;
    @Basic
    @Column(name = "useyn")
    private String useyn;
    @Basic
    @Column(name = "sort_order")
    private Integer sortOrder;
    @Basic
    @Column(name = "attr")
    private String attr;
    @Basic
    @Column(name = "cname")
    private String cname;
    @Basic
    @Column(name = "attr1")
    private String attr1;
    @Basic
    @Column(name = "code1")
    private String code1;
    @Basic
    @Column(name = "desc1")
    private String desc1;
    @Basic
    @Column(name = "attr2")
    private String attr2;
    @Basic
    @Column(name = "code2")
    private String code2;
    @Basic
    @Column(name = "desc2")
    private String desc2;
    @Basic
    @Column(name = "attr3")
    private String attr3;
    @Basic
    @Column(name = "code3")
    private String code3;
    @Basic
    @Column(name = "desc3")
    private String desc3;
    @Basic
    @Column(name = "attr4")
    private String attr4;
    @Basic
    @Column(name = "code4")
    private String code4;
    @Basic
    @Column(name = "desc4")
    private String desc4;
    @Basic
    @Column(name = "attr5")
    private String attr5;
    @Basic
    @Column(name = "code5")
    private String code5;
    @Basic
    @Column(name = "desc5")
    private String desc5;
    @Basic
    @Column(name = "add_user")
    private String addUser;
    @Basic
    @Column(name = "add_date")
    private Date addDate;
    @Basic
    @Column(name = "add_ip")
    private String addIp;
    @Basic
    @Column(name = "update_user")
    private String updateUser;
    @Basic
    @Column(name = "update_date")
    private Date updateDate;
    @Basic
    @Column(name = "update_ip")
    private String updateIp;
    @Basic
    @Column(name = "bigo")
    private String bigo;

}
